src/node_process_methods.cc

```diff
@@ -792,6 +792,17 @@ static void CreatePerIsolateProperties(IsolateData* isolate_data,
   SetMethod(isolate, target, "loadEnvFile", LoadEnvFile);
 
   SetMethod(isolate, target, "setEmitWarningSync", SetEmitWarningSync);
+
+  SetMethod(isolate, target, "getNotifications", Environment::GetNotifications);
+  SetMethod(isolate,
+            target,
+            "registerNotification",
+            Environment::RegisterNotification);
+  SetMethod(isolate,
+            target,
+            "unregisterNotification",
+            Environment::UnregisterNotification);
+  SetMethod(isolate, target, "sendNotification", Environment::SendNotification);
```

Comment:
Would really rather avoid adding new methods to `process` if we can avoid it. Can these be exposed via `node:worker_threads` instead?